<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="PasswordLineEdit" name="MasterPassword_input">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>791</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="TableWidget" name="MainTable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>791</width>
     <height>391</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Machine name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>IP address</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Action</string>
    </property>
   </column>
  </widget>
  <widget class="PrimaryPushButton" name="FetchPasswords_Button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>791</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Fetch passwords</string>
   </property>
  </widget>
  <widget class="TextEdit" name="output_TextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>540</y>
     <width>791</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="PushButton" name="Flush_TextEdit">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>730</y>
     <width>102</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Flush</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PrimaryPushButton</class>
   <extends>PushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>PasswordLineEdit</class>
   <extends>LineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
